docker-compose.production.yml

```diff
@@ -21,6 +21,8 @@ services:
       - "DATABASE_URL=postgres://postgres@postgres/postgres"
       - "SECRET_KEY=fdshjkfds"
       - "ALLOWED_HOSTS=localhost"
+      # We can't expose mailgun api key safely to cypress so using fake email backend
+      - "MAIL_BACKEND=django.core.mail.backends.console.EmailBackend"
```

Comment:
Can't console be the default? Preusmably that's what you'd want for development too.

Reply:
It already is: https://github.com/embassynetwork/modernomad/blob/master/modernomad/settings/local.py#L10

I guess one could argue that it should only be an environment variable. At least it'll be one less environment variable that you need to remember to set.

Reply:
Ah, I see. One pattern I used on the S3 stuff is to enable the S3 storage engine if API keys are set. That way there aren't any superfluous environment variables - the presence of the API keys enables it.

Reply:
That's fair :)

Reply:
Changed to that approach.